src/main/java/frc/robot/commands/RunWinch.java

```diff
@@ -0,0 +1,52 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+import frc.robot.Robot;
+import java.util.function.DoubleSupplier;
+import frc.robot.subsystems.Winch;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+public class RunShoulder extends CommandBase {
+
+  DoubleSupplier climb;
+  public static CANSparkMax shoulder;
+  /**
+   * Creates a new RunClimber.
+   */
+  public RunShoulder(DoubleSupplier climb) {
+    this.climb = climb;
+    
+    shoulder = new CANSparkMax(11,MotorType.kBrushless);
```

Comment:
You put this in subsystems/Climber.java now. It shouldn't be here anymore (and it would be wrong anyway, as you're using the Shoulder CAN ID here).